README.old.md

```diff
@@ -0,0 +1 @@
+# coding-monads-starter
```

Comment:
This file is definitely necessary, because newer version of this file was listed above.